exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -139,52 +143,74 @@ impl CommandExtension for Run {
     fn args(&self) -> Vec<Argument> {
         vec![
             Argument::new_named(
-                EJB_JVM_ARGUMENTS,
+                JVM_DEBUG_SOCKET,
                 false,
-                "Additional parameters for JVM. Must not have a leading dash. \
-                 For example, `Xmx2G` or `Xdebug`",
+                "Allows JVM being remotely debugged. Takes a socket address as a parameter in form \
+                of `HOSTNAME:PORT`. For example, `localhost:8000`",
                 None,
-                "ejb-jvm-args",
-                true,
+                "jvm-debug",
+                false,
             ),
             Argument::new_named(
-                EJB_JVM_DEBUG_SOCKET,
+                JVM_ARGS_PREPEND,
                 false,
-                "Allows JVM being remotely debugged. Takes a socket address as a parameter in form \
-                of `HOSTNAME:PORT`. Must not have a leading dash. For example, `localhost:8000`",
+                "Additional parameters for JVM that prepend the rest of arguments. Must not have a \
+                leading dash. For example, `Xmx2G` or `Xdebug`",
                 None,
-                "jvm-debug",
+                "jvm-args-prepend",
+                true,
+            ),
+            Argument::new_named(
+                JVM_ARGS_APPEND,
                 false,
+                "Additional parameters for JVM that get appended to the rest of arguments. Must not\
+                 have a leading dash. For example, `Xmx2G` or `Xdebug`",
+                None,
+                "jvm-args-append",
+                true,
             ),
         ]
     }
 
     fn execute(&self, mut context: Context) -> Result<Context, failure::Error> {
-        let user_parameters: Vec<String> =
-            context.arg_multiple(EJB_JVM_ARGUMENTS).unwrap_or_default();
-        let jvm_debug_socket = context.arg(EJB_JVM_DEBUG_SOCKET).ok();
+        let args_prepend: Vec<String> =
+            context.arg_multiple(JVM_ARGS_PREPEND).unwrap_or_default();
+        let args_append: Vec<String> =
+            context.arg_multiple(JVM_ARGS_APPEND).unwrap_or_default();
+        let jvm_debug_socket = context.arg(JVM_DEBUG_SOCKET).ok();
+
+        let jvm_config = JvmConfig {
+            args_prepend,
+            args_append,
+            jvm_debug_socket,
+        };
 
-        let curr_config: Config = context
+        let svc_config: BTreeMap<String, Value> = context
```

Comment:
What is svc?

Reply:
This is a shortening for `service` (meaning ejb service). There are `service_config` and `config` bindings for `ServiceConfig` and `Config` instances a few lines below in code.